chemie/chemie/settings/base.py

```diff
@@ -112,8 +113,15 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
```

Comment:
This is already included in the master branch. Ensure that you are up to date with the master branch to prevent merge conflicts (run `git merge origin/master` on your branch)